pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -260,6 +255,10 @@ func (l *podNICImpl) PlugPhase2(vmi *v1.VirtualMachineInstance, iface *v1.Interf
 	}
 
 	pid := "self"
+	bindMechanism, err := l.getPhase2Binding(vmi, iface, network, domain, podInterfaceName, nil)
```

Comment:
not the right place for `pid := "self"`

Reply:
Would you point the correct place for me ? afaiu, the first call for bindmechanism required the PID. 

Which one do you want first ?

Reply:
My bad.